dockers/docker-platform-monitor/docker_init.j2

```diff
@@ -59,8 +59,8 @@ if [ $? -ne 0 ]; then
 fi
 
 # If the Python 3 sonic-platform package is not installed, try to install it
-python3 -c "import sonic_platform" > /dev/null 2>&1 || pip3 show sonic-platform > /dev/null 2>&1
-if [ $? -ne 0 ]; then
+PYTHON3_PLATFORM_PACKAGE=/usr/local/lib/python3.7/dist-packages/sonic_platform
```

Comment:
The python minor version might change. Suggest to use pkgutil.

Reply:
Same comment as for Python2.